mods/default/nodes.lua

```diff
@@ -1523,6 +1523,13 @@ minetest.register_node("default:sand_with_kelp", {
 	inventory_image = "default_kelp.png",
 	paramtype2 = "leveled",
 	groups = {snappy = 3},
+	selection_box = {
+		type = "fixed",
+		fixed = {
+				{-0.5, -0.5, -0.5, 0.5, 0.5, 0.5},
+				{-0.1, 0.5, -0.1, 0.1, 3.5, 0.1},
```

Comment:
Could this be pixel-aligned for consistency with other plant selectionboxes? So -2 / 16 to 2 / 16 horizontally.

Reply:
Even better:
{-0.125, 0.5, -0.125, 0.125, 3.5, 0.125},

Reply:
Done. I don't like it being this large, but I doubt anyone should mind too much.